secret-enum-types.c/h.template: Fix reproducibility issue 

When full filenames are used in generated sources it makes the resulting
debug packages non-reproducible. Best practise is to use basename in
comments instead.

Signed-off-by: Richard Purdie richard.purdie@linuxfoundation.org
Upstream-Status: Submitted [https://gitlab.gnome.org/GNOME/libsecret/-/merge_requests/68]

Index: a/libsecret/secret-enum-types.c.template
===================================================================
--- a/libsecret/secret-enum-types.c.template
+++ b/libsecret/secret-enum-types.c.template
@@ -24,7 +24,7 @@
 
 /*** BEGIN file-production ***/
 #include "@filename@"
-/* enumerations from "@filename@" */
+/* enumerations from "@basename@" */
 /*** END file-production ***/
 
 /*** BEGIN value-header ***/
Index: a/libsecret/secret-enum-types.h.template
===================================================================
--- a/libsecret/secret-enum-types.h.template
+++ b/libsecret/secret-enum-types.h.template
@@ -27,7 +27,7 @@ G_BEGIN_DECLS
 
 /*** BEGIN file-production ***/
 
-/* enumerations from "@filename@" */
+/* enumerations from "@basename@" */
 /*** END file-production ***/
 
 /*** BEGIN value-header ***/
